addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/PiiScanRuleUnitTest.java

```diff
@@ -364,6 +368,65 @@ void shouldAlertWhenResponseSeemsToBeAPdfFileAndLookingForPdfsAtLowThreshold()
         assertThat(alertsRaised.get(0).getEvidence(), is(equalTo(ccNum)));
     }
 
+    @Test
+    void shouldNotRaiseAlertWhenPiiIsInJavascriptContentTypeOnDefaultThreshold() throws Exception {
+        // Given
+        String ccNum = "4111 1111 1111 1111";
+        HttpMessage msg = createMsg(ccNum);
+        msg.getRequestHeader().setURI(new URI("http://example.com/generate/", true));
+        msg.getResponseHeader()
+                .setHeader(HttpResponseHeader.CONTENT_TYPE, "application/javascript");
+        rule.setAlertThreshold(AlertThreshold.DEFAULT);
```

Comment:
These should be parameterized tests that use both MEDIUM and HIGH (not default).

Reply:
Done